notebooks/04_model_select_and_optimize/aml_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>I would add a conclusion here. The objective of using hyperparameter tuning is to have show that the performance metrics are better than a non-optimized selection of parameters. I think we should do a comparison here on the metrics obtained on this notebook with the metrics obtained in the plan quick start notebook of W&amp;D</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Added another wide-deep model results trained with known-hyperparameters in google's [movielens example](https://github.com/tensorflow/models/tree/master/official/wide_deep). Also added conclusion.

Btw, our quickstart notebook uses the hyperparameter we found from this notebook.